.travis.yml

```diff
@@ -57,6 +51,7 @@ install:
   - for dir in kafka examples ; do (cd $dir && go install ${BUILD_TYPE} ./...) ; done
 
 script:
-  - if [[ -f .do_lint ]]; then golint -set_exit_status ./... ; fi
+  # should be replaced with golangci-lint
```

Comment:
why?

Reply:
Its much more configurable, sometimes golint not suitable and you want to run other linters to other folders such as:  examples.